paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -237,5 +237,15 @@ class MKLDNNHandler {
   bool is_reusing_;
 };
 
+inline mkldnn::memory::format MKLDNNFormatForSize(
```

Comment:
There is a same function at here

https://github.com/PaddlePaddle/Paddle/blob/066f537ad2f446da670e861159adbba26fd0c568/paddle/fluid/framework/data_layout_transform.h#L65-L70

Maybe you can directly remove the same one.

Reply:
Done.